File: AR-VR/tanya007_Tanya_2024ec1197_2/gettingStarted/gradingSystem.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cin>>n;
    if(n>90){
        cout<<"excellent";
    }
    else if(n>80 & n<=90){
        cout<<"good";
    }
    else if(n>70 & n<=80){
        cout<<"fair";
    }
    else if(n>60 & n<=70){
        cout<<"meets expectations";
    }
    else{
        cout<<"below par";
    }
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/GradingSystem.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      // input - don't change this code
      Scanner scn = new Scanner(System.in);
      int marks = scn.nextInt();
      // input - don't change this code
      if(marks>90){
      System.out.print("excellent");}
      else if(marks>80 && marks<=90){
        System.out.print("good");}
      else if(marks>70 && marks<=80){
        System.out.print("fair");}
      else if(marks>60 && marks<=70){
        System.out.print("meets expectations");}
      else if(marks<=60){
        System.out.print("below par");}
      // code here      
  
   }
  }